compute/vm-image-spec-bookworm.yaml

```diff
@@ -0,0 +1,126 @@
+# Supplemental file for neondatabase/autoscaling's vm-builder, for producing the VM compute image.
```

Comment:
Maybe this is a candidate for jsonnet in the future if both the spec files share most of their contents

Reply:
Maybe.
For Debian 12 we don't need to build cgroups v2 from sources as it's available in apt repo.